src/seedu/addressbook/data/person/Person.java

```diff
@@ -83,5 +83,18 @@ public int hashCode() {
     public String toString() {
         return getAsTextShowAll();
     }
+    
+    public String toPrintableString () {
+        return getPrintableString(this.getName(), this.getPhone(), this.getEmail(), this.getAddress());
+    }
+    
+    public String getPrintableString (Printable... printables) {
+        final StringBuilder builder = new StringBuilder();
```

Comment:
Good job using StringBuilder. You may also wish to take a look at the java StringJoiner and String.join classes